### PR TITLE
ENH added the change log

### PR DESCRIPTION
**Pull request by beckermr**
This PR adds the changelog.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+
+### removed
+ - Dropped support for Gaussian fits.
+ - Dropped support for ngmix 1.0
+
+
+## 0.3.10 - 2021-10-02
+
+### changed
+ - Refactored LSST code to a functional approach.
```
Comment (esheldon) [inline CHANGELOG.md:53]: Note the code isn't "functional" in a technical sense, it is what experts would call imperative/procedural.

https://en.wikipedia.org/wiki/Functional_programming
https://en.wikipedia.org/wiki/Imperative_programming

**Review by esheldon — COMMENTED** [review]
Thanks this is great.  This is going above and beyond, trolling through the morass of old commit messages.

File: CHANGELOG.md (hunk 1 above)
Comment (beckermr) [reply CHANGELOG.md:53]: I never follow all of these things. :(

Comment (beckermr) [inline CHANGELOG.md:53]:
```suggestion
 - Refactored LSST code to use functions instead of classes.
```

**Pull request merged**